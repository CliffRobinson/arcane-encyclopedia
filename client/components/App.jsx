import React from "react";
import Cards from "./Cards";
import ManaSelector from "./ManaSelector";
import SearchSelector from "./SearchSelector";
import FilterBar from "./FilterBar";


const App = () => {
    return (
        <div className="App" >
            <div className="topTwo">
                <SearchSelector />
                <FilterBar />
            </div>

            <div className="flexContain">
                <div className="main">
                    <Cards />
                </div>
                <div className="sideColumn">
                    <ManaSelector />
                </div>
                
            </div>
        </div>
    );
};

export default App;

