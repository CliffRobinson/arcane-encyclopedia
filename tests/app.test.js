import React from "react";
import {shallow} from "enzyme";

import App from "../client/components/App";

test("Test Suite Running", () => {
    expect(true).toBeTruthy();
});

test("App matches snapshot", ()=> {
    expect(
        toJson(shallow(<App />))
    ).toMatchSnapshot();
});