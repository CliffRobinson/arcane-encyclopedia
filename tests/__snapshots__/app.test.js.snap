// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<div
  className="App"
>
  <div
    className="topTwo"
  >
    <Connect(SearchSelector) />
    <Connect(FilterBar) />
  </div>
  <div
    className="flexContain"
  >
    <div
      className="main"
    >
      <Connect(Cards) />
    </div>
    <div>
      <Connect(ManaSelector) />
    </div>
  </div>
</div>
`;
